middlewares/authn.js

```diff
@@ -17,6 +17,12 @@ module.exports = function(req, res, next) {
         return;
     }
 
+    if (/^\/(pl)$/.test(req.path) && config.authType !== 'none') {
```

Comment:
```suggestion
    if (/^\/(pl)$/.test(req.path)) {
```
@mwest1066 I'm not sure if this check is needed after all

Reply:
Actually I think this whole check is wrong. I think we don't want to bypass auth for `/`, because we want to check for auth cookies below so that we can show an authenticated home page (if we have a cookie).

I think we also need to edit the code below where it says `// if authn cookie check failed then clear the cookie and redirect to login`. That `if` statement should be something like:

```
if (authnData == null && !/^\/(pl)$/.test(req.path)) {
    // if we aren't authenticated then redirect to the login page, unless we are trying to get to the home page which allows access without auth
    ...
```

(we can drop the `authn_provider_name` check in this `if` statement because it's old)

Reply:
Are you saying this is all we need: cb5688aafe86b009e562cab7f13e47f072b20932